openstates/mo/events.py

```diff
@@ -0,0 +1,155 @@
+import re
+import datetime
+import requests
+
+from lxml import html
+from openstates.utils import LXMLMixin
+from billy.scrape.events import EventScraper, Event
+
+
+
+class MOEventScraper(EventScraper, LXMLMixin):
+    jurisdiction = 'mo'
+
+    keywords = ["Committee:", "Chair:", "Vice Chair:", "Time:", "Location:", "Note:"]
```

Comment:
Nice -- like seeing all these constant strings in one place since they are most likely to drift.